src/test/regress/expected/multi_insert_select.out

```diff
@@ -3242,5 +3242,209 @@ returning text_col_1;
  string
 (1 row)
 
+CREATE TABLE dist_table_3(
+dist_col bigint,
+int_col integer
+);
+SELECT create_distributed_table('dist_table_3', 'dist_col');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- dist_table_2 and dist_table_3 are non-colocated source tables. Pulling the SELECT part to coordinator is also not possible.
```

Comment:
improve comment, the below is also similar but that works. what is the difference? Different data types cannot be repartitioned, this is more than non-colocated. Strictly non-colocated :)